scripts/utils/generate_ts.sh

```diff
@@ -21,82 +21,101 @@ mkdir -p cache || die
+HEADERS += \$\$files(../../src/apps/$project/*.h, true)
 HEADERS += \$\$files(../../nebula/*.h, true)
 
 SOURCES += i18nstrings_p.cpp
 SOURCES += ../i18nstrings.cpp
 SOURCES += \$\$files(../../src/shared/*.cpp, true)
-SOURCES += \$\$files(../../src/apps/vpn/*.cpp, true)
+SOURCES += \$\$files(../../src/apps/$project/*.cpp, true)
 SOURCES += \$\$files(../../nebula/*.cpp, true)
 
 TRANSLATIONS += translations.ts
 
 RESOURCES += \$\$files(../../src/shared/*.qrc, true)
-RESOURCES += \$\$files(../../src/apps/vpn/*.qrc, true)
+RESOURCES += \$\$files(../../src/apps/$project/*.qrc, true)
 RESOURCES += \$\$files(../../nebula/*.qrc, true)
 EOF
-print G "done"
+  print G "done"
 
-QT_HOST_BINS=$(qmake6 -query QT_HOST_BINS)
+  QT_HOST_BINS=$(qmake6 -query QT_HOST_BINS)
 
-print Y "Generating the main translation file... "
-${QT_HOST_BINS}/lupdate translations/generated/dummy_ts.pro -ts translations.ts || die
+  print Y "$project - Generating the main translation file... "
+  ${QT_HOST_BINS}/lupdate translations/generated/$project/dummy_ts.pro -ts translations.ts || die
 
-printn Y "Generating strings for addons... "
-python scripts/addon/generate_all.py
-mkdir -p addon_ts || die
-cp addons/generated/addons/*.ts addon_ts
-print G "done."
+  printn Y "$project - Generating strings for addons... "
+  python3 scripts/addon/generate_all.py
+  mkdir -p addon_ts || die
+  cp addons/generated/addons/*.ts addon_ts
+  print G "done."
 
-for branch in $(git branch -r | grep origin/releases); do
-  git checkout $branch &>/dev/null || die
-
-  printn Y "Importing main strings from $branch..."
-  python cache/generate_strings.py -o translations/generated translations/strings.yaml || die
-  ${QT_HOST_BINS}/lupdate translations/generated/dummy_ts.pro -ts branch.ts || die
-  ${QT_HOST_BINS}/lconvert -i translations.ts branch.ts -o tmp.ts || die
-  mv tmp.ts translations.ts || die
-  rm branch.ts || die
-
-  if [ -f "scripts/addon/generate_all.py" ]; then
-    printn Y "Importing addon strings from $branch..."
-    python scripts/addon/generate_all.py
-    ts_files="addons/generated/addons/*.ts"
-    for f in $ts_files
-    do
-      ts_name=$(basename "$f")
-      if [ -f "addon_ts/${ts_name}" ]; then
-        printn Y "File ${ts_name} exists, updating with branch strings..."
-        ${QT_HOST_BINS}/lconvert -i "addon_ts/${ts_name}" "addons/generated/addons/${ts_name}" -o tmp.ts || die
-        mv tmp.ts "addon_ts/${ts_name}"
-      else
-        printn Y "File ${ts_name} does not exist, copying over..."
-        cp "addons/generated/addons/${ts_name}" addon_ts/
-      fi
-    done
-    rm addons/generated/addons/*.ts || die
+  for branch in $(git branch -r | grep origin/releases); do
+    git checkout $branch &>/dev/null || die
+
+    printn Y "Importing main strings from $branch..."
+    if [ -f translations/strings.yaml ]; then
+      python3 cache/generate_strings.py -o translations/generated/$project translations/strings.yaml || die
```

Comment:
One the branch the project doesn't exist, so you're extracting strings.yaml from VPN into all files.